test/runtests.py

```diff
@@ -83,7 +83,7 @@
     if sys.platform == 'win32':
         binary = 'Build/VcBuild/bin/{}_{}/ch.exe'.format(arch, flavor)
     else:
-        binary = 'BuildLinux/{0}/ch'.format(flavor)
+        binary = 'Build/clang_build/{}_{}/bin/ch'.format(arch, flavor.lower())
     binary = os.path.join(repo_root, binary)
 if not os.path.isfile(binary):
     print('{} not found. Did you run ./build.sh already?'.format(binary))
```

Comment:
>Did you run ./build.sh already? [](start = 25, length = 31)

I prefer to phrase these things with a specific call to action instead of an ambiguous question. I'd prefer "Please ensure you have already run ./build.sh."